config.json.example

```diff
@@ -3,6 +3,9 @@
 		"teams": ["Cubs"],
 		"divisions": ["NL Central"]
 	},
+	"game": {
+		"show_hits_errors": false
+	},
```

Comment:
Part of me is wondering if `rotation` should be a sub-grouping under game (or 'games')? It would break existing config and require a lot of changes to the config class' loading routine. What do you think?

Reply:
The configuration logic is very simple - everything just gets dumped into variables in config/__init__.py.  So moving things around would not be complicated.  It might break people's custom configurations though, if anyone is using this code.

Reply:
Yeah certainly not complicated, just a bunch of tiny changes. It also wouldn't be crazy to update an existing config, but for now I don't think it's worth it. Something to think about in the future